src/Organizer/EditOrganizerRestController.php

```diff
@@ -113,13 +114,29 @@ public function updateUrl($organizerId, Request $request)
     }
 
     /**
-     * @param $organizerId
+     * @deprecated Use updateName with language parameter instead.
+     * @param string $organizerId
      * @param Request $request
      * @return JsonResponse
-     * @throws DataValidationException
      */
-    public function updateName($organizerId, Request $request)
-    {
+    public function updateNameDeprecated(
+        $organizerId,
+        Request $request
+    ) {
+        return $this->updateName($organizerId, 'nl', $request);
+    }
+
+    /**
+     * @param string $organizerId
+     * @param string $language
+     * @param Request $request
+     * @return JsonResponse
+     */
+    public function updateName(
```

Comment:
Goed dat je hier `updateName` gebruikt cfr. de JSON-LD property `name`, en in de domain code `Title` cfr. de ubiquitous language. 👍